Doc/tutorial/errors.rst

```diff
@@ -299,10 +299,9 @@ This can be useful when you are transforming exceptions. For example::
       File "<stdin>", line 4, in <module>
     RuntimeError
 
-The expression following the :keyword:`from` must be either an exception or
-``None``. Exception chaining happens automatically when an exception is raised
-inside an exception handler or :keyword:`finally` section. Exception chaining
-can be disabled by using ``from None`` idiom:
+Exception chaining happens automatically when an exception is raised inside an
+exception handler or :keyword:`finally` section. Exception chaining can be
```

Comment:
What do you think of adding: ```an exception handler (a :keyboard:`try` block)```

Reply:
Or possibly ``a :keyboard `try` block's exception handler``

Reply:
I used a parenthetical on purpose to signify that «exception handler» == «except block» (I wrote `try block` by mistaked at first).
Otherwise it could be confusing what an exception handler is (is it a function you call in except? is it excepthook?)

Reply:
We already use ``:keyword:`finally` section``.
How about just ``:keyword:`except` section`` for consistency?